android/build.gradle

```diff
@@ -35,8 +35,8 @@ android {
 rootProject.allprojects {
     repositories {
         google()
-        maven {
-            url 'https://sdk.squareup.com/public/android'
+         maven {
+            url "https://maven.google.com"
         }
```

Comment:
This might be redundant since you already have `google()`

Reply:
removed it